content/working_with/console/widgets/topology.md

```diff
@@ -4,18 +4,31 @@ title: Topology
 category: Cloudify Console
 draft: false
 ---
+
 Displays the topology of a selected blueprint or deployment.
+
 The blueprint or deployment ID must be selected in one of the following ways: 
 
 * By placing the widget in the blueprints/deployments drill-down page, meaning the blueprint/deployment has been selected before entering the page, and its id is included in the page’s context. 
 * By adding to the page a widget allowing to select blueprints or deployments, such as the resources filter, the blueprints list or the blueprint deployments.  
 
-![show-topology]( /images/ui/widgets/show-topology.png )
+![Topology]( /images/ui/widgets/topology.png )
 
-When executing a `Workflow` for a `Deployment` (e.g. the `install` workflow), the topology nodes show badges that reflect the workflow execution state.<br/>
-    
+## Features
+
+### Presentation
+
+Each of the application's nodes is displayed as a square container that can contain other nodes. 
```

Comment:
Isn't it misleading or inconsistent with the rest of the docs to use "application" term in exchange for blueprint or deployment?

Reply:
"Each of the blueprint's nodes"